### PR TITLE
2021-12-03-nerdl_fewnerd_100d_pipeline_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

**Comment by murat-gunay** [comment]
@josejuanmartinez 
I have chosen the title of the pipeline in consistent with the other [Few-NERD pipeline](https://nlp.johnsnowlabs.com/2021/09/28/nerdl_fewnerd_subentity_100d_pipeline_en.html). consider this when reviewing it

File: docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained pipeline is built on Few-NERD model and it detects :
+
+`PERSON`, ORGANIZATION`, `LOCATION`, `ART`, `BUILDING`, `PRODUCT`, `EVENT`, `OTHER`
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md:22]: Missing quote before ORGANIZATION

File: docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+layout: model
+title: Pre-trained Pipeline for Few-NERD-General NER Model
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md:3]: `Pretrained pipeline for the Few-NERD-General model`

Comment (josejuanmartinez) [reply docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md:3]: Preventing technical details is for demos. In ModelsHub pages we should also be careful, but I think this is ok.

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
2 little changes and we are good to go.

File: docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md]: @josejuanmartinez 
thanks for your feedback. I fixed and committed the changes. You can approve the review.

File: docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md (hunk 2 above)
Comment (josejuanmartinez) [reply docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md:3]: `Pretrained`, without hyphen. Prefixes are not hyphenated by  default:
image

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Don't separate prefixes by hyphen. Take a look. The rest is ok. Thank you!

File: docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+</div>
+
+## Results
+
+```bash
++-----------------------+------------+
+|chunk                  |ner_label   |
++-----------------------+------------+
+|Kentucky Fried Chicken |ORGANIZATION|
+|KFC                    |BUILDING    |
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md:54]: KFC is not building I think

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
KFC has a false positive. Can we fix it?

**Review by josejuanmartinez — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**